app/mailers/patient_list_download_mailer.rb

```diff
@@ -0,0 +1,18 @@
+class PatientListDownloadMailer < ApplicationMailer
+  default from: 'help@simple.org'
+
+  def patient_list(recipient_email, model_type, model_name, patients_csv)
+    @model_type = model_type
+    @model_name = model_name
```

Comment:
Is marking them as instance variables necessary here?

Reply:
Yes, so they make it into the view

Reply:
Makes sense.